<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(251,102,102, 200), stop:1 rgba(20,196,188, 210))</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>631</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Bright</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>STL Position Translator</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>1. Chose the original stl File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadstl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius: 20px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
border-radius: 20px;
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
border-radius: 20px;
}
</string>
   </property>
   <property name="text">
    <string>Chose a single stl file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>2. Chose The position data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadposition">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>560</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius: 20px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
border-radius: 20px;
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
border-radius: 20px;
}
</string>
   </property>
   <property name="text">
    <string> Chose the position data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>630</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>3. Generate the new stl file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>660</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius: 20px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
border-radius: 20px;
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
border-radius: 20px;
}
</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEditor">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Please chose a single .stl file and the order that corresponds to it</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEditor_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>Please chose your position data file</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextEditor_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>640</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>440</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>Location Nr.</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>420</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;; background-color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>Setting:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="setting">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <item>
    <property name="text">
     <string>Base on the Center of Gravity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Base on the Bounding Box</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Base on the Center of Bounding Box</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="setting_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>520</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <item>
    <property name="text">
     <string>.xlsx</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.txt</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_widget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadstl_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius: 20px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
border-radius: 20px;
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
border-radius: 20px;
}
</string>
   </property>
   <property name="text">
    <string>Chose a group of stl files</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(255 , 0 , 0);
	border-radius: 20px;
}

QPushButton:hover
{
	background-color: rgb(220, 0, 0);
border-radius: 20px;
}

QPushButton:pressed
{
	background-color:rgb(200 , 20 , 20);
	padding-left:3px;
	padding-top:3px;
border-radius: 20px;
}
</string>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
